### PR TITLE
Proposal to initialize Base cUSDbCv3 market

### PR DESCRIPTION
**Pull request by kevincheng96**
The proposal to initialize the cUSDbCv3 market.

### CODE REVIEW

File: deployments/base/usdc/migrations/1689892563_configurate_and_ens.ts [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        signature: '_setCompSpeeds(address[],uint256[],uint256[])',
+        args: [
+          [cUSDCAddress],
+          [10583333330000000n],
+          [15444444444444444n],
+        ],
+      },
+    ];
+
+    const description = "TODO"; // TODO
```
Comment (kevincheng96) [inline deployments/base/usdc/migrations/1689892563_configurate_and_ens.ts:146]: WIP

Comment (kevincheng96) [reply deployments/base/usdc/migrations/1689892563_configurate_and_ens.ts]: Done

File: deployments/base/usdbc/migrations/1689892563_configurate_and_ens.ts [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+      USDbC
+    } = await deploymentManager.getContracts();
+
+    const {
+      comptrollerV2,
+    } = await govDeploymentManager.getContracts();
+
+    // 1.
+    const stateChanges = await diffState(comet, getCometConfig, preMigrationBlockNumber);
+    // TODO: uncomment when contracts are deployed
```
Comment (scott-silver) [inline deployments/base/usdbc/migrations/1689892563_configurate_and_ens.ts:174]: just a reminder to uncomment out this test before merging

**Review by scott-silver — APPROVED** [review]
👍

File: deployments/base/usdbc/migrations/1689892563_configurate_and_ens.ts (hunk 2 above)
Comment (kevincheng96) [reply deployments/base/usdbc/migrations/1689892563_configurate_and_ens.ts]: Uncommented

**Comment by kevincheng96** [comment]
Migration script was enacted in this Github Actions run: https://github.com/compound-finance/comet/actions/runs/5767267486/job/15636682979

**Pull request merged**